### PR TITLE
Fix `copy` command

### PR DESCRIPTION
**Pull request by adiee5**
Based on #11

### CODE REVIEW

File: src/disk_commands.p8 [hunk 1]
```diff
@@ -277,7 +273,7 @@ io_error:
         if_cs {
             newfilename = main.command_arguments_ptr + space_idx + 1
             main.command_arguments_ptr[space_idx] = 0
-            diskio.list_filename[0] = 'c'
+            diskio.list_filename[0] = petscii:'c'
             diskio.list_filename[1] = ':'
```
Comment (irmen) [inline src/disk_commands.p8:277]: for consistency, please also add petscii: prefix to the other 2 characters (even though the resulting byte value will be the same)

**Review by irmen — CHANGES_REQUESTED** [review]

File: src/disk_commands.p8 (hunk 1 above)
Comment (adiee5) [reply src/disk_commands.p8:277]: Ok, will do, when i get back home

Comment (adiee5) [reply src/disk_commands.p8]: @irmen

**Comment by irmen** [comment]
thanks for fixing this!

**Pull request merged**